app/controllers/api/v0/users/games_controller.rb

```diff
@@ -25,13 +25,13 @@ def invite_rsvp
     if user_game
       if params[:rsvp] == "accept"
         user_game.update(invitation: 1)
-        render json: { data: { message: "Successfully Accepted Invitation"}}, status: 201
```

Comment:
This is another example of where I liked the previous version better than the original. Having white space allows your brain to be able to separate the values from their surroundings and process the data more quickly.

Reply:
I am unsure of what lines you are referring to specifically.